### PR TITLE
BXMSDOC-8106 updated getting started with case management

### PR DESCRIPTION
**Pull request by emmurphy1**

### CODE REVIEW

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-creating-manager-approval-proc.adoc [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
+. In the *IT_Orders_New* project, click the *orderhardware business processes*.
 . Create and configure the `Prepare hardware spec` user task:
-.. Expand *Tasks* in the *Object Library* and drag a user task onto the canvas and convert the new task to a user task.
-.. Click the new user task and in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
-.. Input `Prepare hardware spec` in the *Name* field.
-.. Expand *Implementation/Execution*, click *Add* below the *Groups* menu, click *Select* -> *New*, and input `supplier`.
+.. Select the *Activities* icon in the *Object Library* and click *User*.
+.. Select the new user task and in the upper-right corner click the *Properties* image:getting-started/diagram_properties.png[] icon.
+.. Enter `Prepare hardware spec` in the *Task Name* field.
+.. lick *Add* below the *Groups* menu, click *Select* -> *New*, and input `supplier`.
```
Comment (hmanwani-rh) [inline doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-creating-manager-approval-proc.adoc]:
```suggestion
.. Click *Add* below the *Groups* menu, click *Select* -> *New*, and enter `supplier`.
```

**Review by hmanwani-rh — COMMENTED** [review]
@emmurphy1 I've added a few suggestions below. Rest looks good to me.

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-create-sub-proc.adoc [hunk 2]
```diff
@@ -35,17 +37,17 @@ image::cases/place-order-io1.png[]
...
 
 . Click *OK*.
-. Select the *Skippable* check box and enter the following text in the *Description* field:
+. In the *Properties* panel, select the *Skippable* check box and enter the following text in the *Description* field:
 +
 `Approved order #{CaseId} to be placed`
 +
 image::cases/skip-description.png[]
-. Create an outgoing connection from the *Place order* user task and connect it to an end event.
+. Click the *Place order* user task and then click the *Create End* icon to create an outgoing connection from the *Place order* user task and connect it to an end event.
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-create-sub-proc.adoc]: Good call!

File: doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-creating-a-case-proc.adoc [hunk 3]
```diff
@@ -10,15 +10,14 @@ Create a new `IT_Orders` project in {CENTRAL} to gain an understanding of all of
 +
 .Default spaces
 image::getting-started/spaces-teams-bc.png[]
-. Click the *Add Project* drop-down arrow and select the *Case project* option:
+. Click the arrow next to *Add Project* and select *Case project*:
```
[the remark this answers is not in the document]
Comment (emmurphy1) [reply doc-content/jbpm-docs/src/main/asciidoc/GettingStartedCaseManagement/case-management-creating-a-case-proc.adoc:13]: Rejected. drop-down is not necessary here, and in any case the arrow isn't a drop-down, the menu is.

**Pull request merged**